### DISCUSSION TITLE
Provide a button to toggle /// comments

### DISCUSSION THREAD

**Discussion by suugbut** [community/community#78525] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Most of the time I feel that the presence of /// comments just add unnecessary distraction to the code I am deciphering. 
Especially when I already knows the purpose of the methods to which the /// comments are added.
So if  possible please kindly provide a button to toggle (show/hide) all the /// comments in the code we are reading.

Or providing more general options to toggle (hide/show):
- `[]` attributes
- `///` comments
- `//` comments
-  line numbers
- etc

**Comment by vishaltechie21** [comment]
Feature Request: If you're using a specific code editor or IDE, you might consider submitting a feature request to the developers. Ask for an option to toggle the visibility of comments, line numbers, or other elements directly within the editor interface. Many software developers appreciate user feedback and might consider implementing such a feature in future updates.

Customization Options: Explore the settings/preferences within your code editor. Some editors offer customization options that allow you to hide/show specific elements. Look for settings related to comments, line numbers, or other aspects of code display. Not all editors have these options, but it's worth checking.

Plugins or Extensions: Check if there are any plugins or extensions available for your code editor that provide the functionality you're seeking. Sometimes, third-party extensions can add features like toggling comment visibility or hiding line numbers.

User Scripts or Custom Solutions: In some cases, users create custom scripts or browser extensions to modify the behavior of code editors. If you're comfortable with scripting or extensions, you might explore creating a custom solution tailored to your preferences. This might involve scripting languages like JavaScript or Python, depending on the editor.

**Reply by suugbut** [reply]
Mostly I am viewing the code on browser and not downloading the repos.
